src/lib/seam/SeamProvider.tsx

```diff
@@ -70,42 +70,40 @@ export function SeamProvider({
   useSeamStyles({ disabled: disableCssInjection, unminified: unminifiyCss })
   useSeamFont({ disabled: disableFontInjection })
 
-  const value = useMemo(() => {
+  const seamProviderContextValue = useMemo(() => {
     const context = createSeamContextValue(props)
     if (
       context.client == null &&
       context.publishableKey == null &&
       context.clientSessionToken == null
     ) {
-      return defaultSeamContextValue
+      return getDefaultSeamContextValue()
```

Comment:
This behavior is slightly different but more correct. If the `globalThis.seam` client is defined after this file is imported it will now correctly grab it

Reply:
Cool. Can you undo the variable rename though, or simply name it `contextValue`? I do not think the longer name is needed in this context.